src/OnlineSales/Controllers/BaseController.cs

```diff
@@ -110,7 +110,7 @@ public virtual async Task<ActionResult> Delete(int id)
         [ProducesResponseType(StatusCodes.Status401Unauthorized)]
         [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status404NotFound)]
         [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
-        public virtual async Task<ActionResult<List<TD>>> Get([FromQuery] IDictionary<string, string>? parameters)
+        public virtual async Task<ActionResult<List<TD>>> Get([FromQuery] string? query)

```

Comment:
is "this.Request.QueryString" doesn't work without declaration "[FromQuery] string? query"?